src/utils/V3Utils.ts

```diff
@@ -83,6 +85,11 @@ export function isSlowFill(fill: Fill): boolean {
   return isV2Fill(fill) ? fill.updatableRelayData.isSlowRelay : fill.relayExecutionInfo.fillType === FillType.SlowFill;
 }
 
+// If needed, this can be made generic in the future.
+export function isSlowFillRequest(fill: Fill | SlowFillRequest): fill is SlowFillRequest {
```

Comment:
ooc, why are we unsure of whether an object is a Fill or a SlowFillRequest? They're very separate events and they aren't mixed within the SpokePoolClient. My initial impression is that it seems unusual to mix them like this.